core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -254,10 +254,10 @@ class TileMap {
                 val containsViewableNeighborThatCanSeeOver = cTile.neighbors.any {
                         bNeighbor: TileInfo ->
                     val bNeighborHeight = bNeighbor.getHeight()
-                    if(cTile.resource=="Marble"
-                        && bNeighbor.terrainFeatures.contains("Forest")
-                    )
-                        println()
+//                    if(cTile.resource=="Marble"
+//                        && bNeighbor.terrainFeatures.contains("Forest")
+//                    )
+//                        println()
```

Comment:
I vote for completely removing this code instead of just commenting it out

Reply:
Yup - another oops. I admit doing conditional breakpoints this way executes faster... But... Well, now you know where all those spurious empty lines on the console came from when you analyzed that masochist's save.